modulemd/v2/include/modulemd-2.0/modulemd-rpm-map-entry.h

```diff
@@ -67,6 +67,34 @@ ModulemdRpmMapEntry *
 modulemd_rpm_map_entry_copy (ModulemdRpmMapEntry *self);
 
 
+/**
+ * modulemd_rpm_map_entry_equals_wrapper:
+ * @a: const void pointer
+ * @b: const void pointer
+ *
+ * Returns: TRUE if the two entries contain equivalent data. FALSE if they
+ * differ.
+ *
+ * Since: 2.5
+ */
+gboolean
+modulemd_rpm_map_entry_equals_wrapper (const void *a, const void *b);
```

Comment:
Do not make the wrapper functions public API. They should be either static functions within the C file that consumes them or else in the headers in the `include/private/` path so that they are visible only to internal consumers. There's no reason to expose these to the consumers of libmodulemd.